ruby/random.rb

```diff
@@ -0,0 +1,15 @@
+def random_group
+  group = ['A', 'B', 'C', 'D', 'E', 'F']
+
+  group1 = if [true, false].sample
+             (group - group.sample(3)).sort
+           else
+             (group - group.sample(4)).sort
+           end
```

Comment:
以下でも同じ結果になりそうな感じがします。

```suggestion
  group1 = (group - group.sample([3,4].sample)).sort
```

Reply:
もっというとこれで十分に見えます。

```ruby
group1 = group.sample([3,4].sample).sort
group2 = group - group1
```